docs/iris/src/whatsnew/2.3.rst

```diff
@@ -0,0 +1,196 @@
+* :meth:`iris.cube.Cube.remove_cell_measure` now also allows removal of a cell
+  measure by its name (previously only accepted a CellMeasure object).
+* The :data:`iris.analysis.RMS` aggregator now supports a lazy calculation.
+  However, the "weights" keyword is not currently supported by this, so a
+  *weighted* calculation will still return a realised result, *and* force
+  realisation of the original cube data. **NOTE:** lazy calculation with RMS
+  is currently only supported for un-masked arrays.
+* Iris now supports NetCDF Climate and Forecast (CF) Metadata Conventions 1.7 (see `CF 1.7 Conventions Document <http://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/cf-conventions.html>`_ for more information)
+
+
+Iris 2.3.0 Dependency Updates
+=============================
+* The Proj4 pin has been moved from version 5 to version 6 pending `fixes to
+  some cartopy tests <https://github
+  .com/SciTools/cartopy/pull/1289#pullrequestreview-272774087>`_.
+* Iris now requires Dask > 1.2 to allow for improved coordinate equality
+  checks.
+
+
+Bugs Fixed
+==========
+* Cube equality of boolean data is now handled correctly.
+* Fixed a bug where cell measures were incorrect after a cube
+  :meth:`~iris.cube.Cube.transpose` operation.  Previously, this resulted in
+  cell-measures that were no longer correctly mapped to the cube dimensions.
+* The :class:`~iris.coords.AuxCoord` disregarded masked points and bounds, as did the :class:`~iris.coords.DimCoord`.
+  Fix permits an :class:`~iris.coords.AuxCoord` to contain masked points/bounds, and a TypeError exception is now
+  raised when attempting to create or set the points/bounds of a :class:`~iris.coords.DimCoord` with masked arrays.
+* :class:`iris.coord_systems.VerticalPerspective` coordinate system now uses
+  the `CF Vertical perspective definition <http://cfconventions
+  .org/cf-conventions/cf-conventions.html#vertical-perspective>`_; had been
+  erroneously using Geostationary.
+* :class:`~iris.coords.CellMethod` will now only use valid `NetCDF name tokens <https://www.unidata.ucar.edu/software/netcdf/docs/netcdf_data_set_components.html#object_name>`_ to reference the coordinates involved in the statistical operation.
+* The following var_names will now only use valid `NetCDF name tokens
+  <https://www.unidata.ucar
+  .edu/software/netcdf/docs/netcdf_data_set_components.html#object_name>`_ to
+  reference the said NetCDF variable name. Note that names with a leading
+  underscore are not permitted.
+    - :attr:`iris.aux_factory.AuxCoordFactory.var_name`
+    - :attr:`iris.coords.CellMeasure.var_name`
+    - :attr:`iris.coords.Coord.var_name`
+    - :attr:`iris.coords.AuxCoord.var_name`
+    - :attr:`iris.cube.Cube.var_name`
+* Cubes are now rendered more robustly by Jupyter.
+  :meth:`~iris.cube.Cube._repr_html_` no longer crashes for a cube with
+  attributes containing ``\n``.
+* Variables which reference themselves in ``cell_measures`` can now be read.
```

Comment:
This is a bit unclear.
I would specify that the you are talking about netCDF here.
Something like:
```suggestion
* NetCDF Variables which reference themselves in ``cell_measures`` can now be read.
```

Reply:
👍